src/os/unix.rs

```diff
@@ -25,6 +27,74 @@ pub use platform::x11;
 pub use platform::XNotSupported;
 pub use platform::x11::util::WindowType as XWindowType;
 
+/// Theme for wayland client side decorations
+///
+/// Colors must be in native endian ARGB8888 format
```

Comment:
Given endianness is handled by SCTK, this comment would probably be clearer without talking about endianness.

Reply:
Yeah, I forgot SCTK now handles it. Also the comments wrong, the colors must only be in big endian anyway.